### PR TITLE
Add ashib

### PR DESCRIPTION
**Pull request by similiken**
Updates ashib tokenomics after token burns.

### CODE REVIEW

**Comment by Backsleepaz** [comment]
@hieupnh  Could you check this?

**Comment by similiken** [comment]
> 
Idk what i'm seeing.
image

File: src/index.ts [hunk 1]
```diff
@@ -99,10 +96,5 @@ export const supplyFetchers: Record<string, SupplyFetcher> = {
     ninjazFetcher,
   "38ad9dc3aec6a2f38e220142b9aa6ade63ebe71f65e7cc2b7d8a8535434c4159":
     clayFetcher,
-  ee0633e757fdd1423220f43688c74678abde1cead7ce265ba8a24fcd43424c50: cblpFetcher,
-  "682fe60c9918842b3323c43b5144bc3d52a23bd2fb81345560d73f634e45574d":
-    newmFetcher,
-  bb4cfbe0f6be60b80f90f815e8353b93431de4df785d75350b9d214a48455242: herbFetcher,
-  "279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f534e454b":
-    snekFetcher,
```
Comment (hieupnh) [inline src/index.ts:107]: Why you delete this block code?

File: src/tokens/ashib.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { defaultFetcherOptions, SupplyFetcher } from "../types";
+
+const ASHIB = "afc910d7a306d20c12903979d4935ae4307241d03245743548e767834153484942";
+
+const fetcher: SupplyFetcher = async (options = defaultFetcherOptions) => {
+    const total = 109000000000;
```
Comment (hieupnh) [inline src/tokens/ashib.ts:6]: You are missing the return of this function

**Comment by hieupnh** [comment]
@Backsleepaz can you check my comment and resolve the conflict in `src/index.ts`

File: src/index.ts (hunk 1 above)
Comment (Backsleepaz) [reply src/index.ts:107]: what do @similiken need to fix to make it work?

**Review by similiken — COMMENTED** [review]
re-adding "import ashibFetcher from "./tokens/ashib";" looks fine.

File: src/tokens/ashib.ts (hunk 2 above)
Comment (similiken) [reply src/tokens/ashib.ts:6]: i guess then let's re-add "export default fetcher;".

Comment (similiken) [reply src/tokens/ashib.ts:6]: Everything appears to exist that needs to exist. If i am missing the return of a function, then i don't know where. everything looks fine to me.

**Pull request closed without merging**